src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -89,8 +89,6 @@ fi
 
 # ========================= BEGIN Test Execution =============================
 echo Running tests... Start time: $(date +"%T")
-echo Commands:
-[[TestRunCommandsEcho]]
```

Comment:
I agree we should try to keep this. Can we not escape the parantheses, or remove them from the echo command?